packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -41,6 +41,8 @@ export interface TabsProps extends Omit<React.HTMLProps<HTMLElement | HTMLDivEle
   isBox?: boolean;
   /** Enables vertical tab styling */
   isVertical?: boolean;
+  /** Enables no border bottom tab styling */
+  hasNoBorderBottom?: boolean;
```

Comment:
Not a blocker but I wonder whether the prop should be called `hasBorderBottom` with a false value instead of `hasNoBorderBottom` with a true value? I know the css prop is called noBorderBottom but I feel like I don't see "no{X}" props as much.

Reply:
I think making it feel more intuitive to the consumer might be better than making it match the core class name. Im in support of avoiding the double negative.